nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -47,6 +47,10 @@ class BtActionNode : public BT::ActionNodeBase
   : BT::ActionNodeBase(xml_tag_name, conf), action_name_(action_name)
```

Comment:
I believe this is missing:
- Goal updater https://github.com/ros-planning/navigation2/blob/main/nav2_behavior_tree/include/nav2_behavior_tree/plugins/decorator/goal_updater_node.hpp#L74
- Battery low https://github.com/ros-planning/navigation2/blob/main/nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/is_battery_low_condition.hpp#L77
- is stuck https://github.com/ros-planning/navigation2/blob/main/nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/is_stuck_condition.hpp#L93